test/image_constants.go

```diff
@@ -19,16 +19,6 @@ package test
 //image_constants.go defines constants that are shared between test-images and conformance tests
 
 const (
-	//EnvImageServerPort is the port on which the environment test-image server starts.
-	// TODO: Modify this port number after https://github.com/knative/serving/issues/2258 is fixed for a stricter verification.
-	EnvImageServerPort = 8080
-
-	//EnvImageEnvVarsPath path exposed by environment test-image to fetch environment variables.
-	EnvImageEnvVarsPath = "/envvars"
-
-	//EnvImageFilePathInfoPath path exposed by environment test-image to fetch information for filepaths
-	EnvImageFilePathInfoPath = "/filepath"
-
-	//EnvImageFilePathQueryParam query param to be used with EnvImageFilePathInfoPath to specify filepath
-	EnvImageFilePathQueryParam = "path"
+	//RuntimeImageServerPort is the port on which the runtime test-image server starts.
+	RuntimeImageServerPort = 8800
```

Comment:
Just one more things... now that this fine is a single constant can we just move it into the test file? I don't see a need for it to be shared. All that matters is that the value specified in the test is the same as the value we are looking for. Other tests can choose any other value.

Reply:
done :slightly_smiling_face: